Add mokosmart LoRaWAN Gateway  MKGW2-LW by allen (#273)
TheThingsIndustries/lorawan-stack-docs: doc/content/gateways/moko-mkgw2-lw/_index.md

---
title: "MOKOSMART MKGW2-LW LoRaWAN Gateway"
description: ""
---

> MOKO 8 channels gateway MKGW2 is based on LoRaWAN protocol, which is embedded with Semtech’s high performance multi-channel transceiver SX1301/SX1257 and MTK platform. It is for

Reviewer: ```suggestion
MOKOSMART's 8 channel MKGW2 gateway is embedded with Semtech’s high performance multi-channel transceiver SX1301/SX1257 and MTK platform. It is for indoor use and is easy to install.
```

> indoor use and easy for installation. 

Reviewer: ```suggestion
```

{{< figure src="mkgw2-lw.jpg" alt="mkgw2-lw" class="plain float" >}}

## Requirements

1. User account on {{% tts %}} with rights to create gateways.

> 2. MOKOSMART MKGW2-LW connected to the internet via ethernet or WIFI.（For the ethernet and wifi setting, you can refer to the  5.login into the Gateway and 6.Internet Setting of the user maunal)

Reviewer: ```suggestion
2. MOKOSMART MKGW2-LW connected to the internet via Ethernet or WiFi. (For Ethernet and WiFi settings, see sections 5 and 6 of the [user manual](https://www.mokosmart.com/lorawan-gateway-mkgw2-lw/))
```

I couldn't find a link to your user manual online, please update this.

## Registration

Create a gateway by following [the instructions]({{< ref "/gateways/adding-gateways" >}}). 

The **EUI** of the gateway can be found on the configuration web page of the gateway. 

{{< figure src="eui.png" alt="Gateway EUI" >}}

> ## Configuration via Browser

Reviewer: Add newline after heading

> 1. Turn On the gateway and waiting for about 60s.

Reviewer: ```suggestion
1. Turn on the gateway and wait for about 60s.
```

> 2. Using your PC or phone connect the SSID of the gateway. The default SSID format is "MKGW2-LW-xxxx” such as " MKGW2-LW-91D8 ", “91D8” is the last two bytes of the gateway MAC address. verify the password (Default: Moko4321) and connect to the gateway.

Reviewer: ```suggestion
2. Using your PC or phone, enter the URL of the gateway in a browser. The default URL format is "MKGW2-LW-xxxx”, e.g "MKGW2-LW-91D8 ", where “91D8” is the last two bytes of the gateway MAC address. The default password is Moko4321.
```

This does not look like a valid url. Are you sure it's just "http://MKGW2-LW-xxxx" ?

> 2. Using your PC or phone connect the SSID of the gateway. The default SSID format is "MKGW2-LW-xxxx” such as " MKGW2-LW-91D8 ", “91D8” is the last two bytes of the gateway MAC address. verify the password (Default: Moko4321) and connect to the gateway.

Reviewer: Ahhh I misunderstood, this is the wireless SSID. Ignore my last comment.

```suggestion
2. Using your PC or phone, connect to the WiFi Access Point exposed by the gateway. The default SSID format is "MKGW2-LW-xxxx”, e.g "MKGW2-LW-91D8 ", where “91D8” is the last two bytes of the gateway MAC address. The default password is Moko4321.
```

3. After successful connection, the WEB management platform can be accessed through the IP address 192.168.22.1 of the gateway LAN interface.

> 4. Log on using the following default credentials, Username: Admin Password: admin

Reviewer: ```suggestion
4. Login using the following default credentials:
- Username: Admin
- Password: admin
```

It is recommended that the default password is changed for security reasons.

{{< figure src="login in web.png" alt="Login In Web" >}}

## Connect to {{% tts %}}

> The MOKOSMART MKGW2-LW  support  UDP packet forwarder. Please follow instructions for connecting the MKGW2-LW with UDP packet forwarder.After completing basic configuration, turn to server address interface of web.

Reviewer: ```suggestion
The MOKOSMART MKGW2-LW supports the UDP packet forwarder. Please follow instructions for connecting the [UDP packet forwarder]({{< ref "gateways/semtech-udp-packet-forwarder" >}}).
```

{{< figure src="eui.png" alt="Gateway EUI" >}}

> Edit the server parameters:

Reviewer: ```suggestion
```

The server settings are all in the UDP Packet Forwarder doc. Remove them here, in case they are updated in the future.

> - **server_address**: Address of the Gateway Server. If you followed the [Getting Started guide]({{< ref "/getting-started" >}}) this is the same as what you use instead of `thethings.example.com`.

Reviewer: ```suggestion
```

Remove these

- **serv_port_up**: UDP upstream port of the Gateway Server, typically 1700.
- **serv_port_down**: UDP downstream port of the Gateway Server, typically 1700.

> Don't foget to click "SAVE&APPLY" after you  fill in the parameters.More information and setting, you can refer to the user manual fo mkgw2-lw from moko sales or send email to support_lora@mokotechnology.com.

Reviewer: ```suggestion
Don't forget to click "SAVE&APPLY" after you fill in the parameters. After a few seconds, your gateway should connect to {{% tts %}}.

For more information, you can refer to the MKGW2-LW user manual or send an email to [support_lora@mokotechnology.com](mailto:support_lora@mokotechnology.com).
```
